IronFoundry.Container.Shared/Win32/SafeHGlobal.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IronFoundry.Container.Win32
+{
+    class SafeHGlobal : SafeBuffer
```

Comment:
This seems very similar to `SafeAllocation` in the `JobObject` p/invoke code.  Can these be combined somehow?  Maybe `SafeAllocation` can inherit from (or use) `SafeHGlobal` ?